PushNotifications/PushNotifications.xcodeproj/project.pbxproj

```diff
@@ -458,6 +458,7 @@
 			hasScannedForEncodings = 0;
 			knownRegions = (
 				en,
+				Base,
```

Comment:
what does this mean?

Reply:
This has been introduced in Xcode 10 and defines base localization. We provide only English at the moment, but since the old way of localizing apps/frameworks is now deprecated, Xcode complains about migration to the new recommended setup.